common/config/rush/pnpm-lock.yaml

```diff
@@ -73,70 +73,70 @@ importers:
       webpack: ^5.1.2
     dependencies:
       '@bentley/icons-generic-webfont': 1.0.34
-      '@itwin/appui-abstract': 4.2.4_@itwin+core-bentley@4.2.4
-      '@itwin/appui-layout-react': 4.7.0_3gdfs22wzwxejstmtxvgxf5eoe
-      '@itwin/appui-react': 4.7.0_ieqmzvdr3ss5mpis4apyxont2y
-      '@itwin/components-react': 4.7.0_lefixqtlpiky2mdodixn2eqa7q
-      '@itwin/core-backend': 4.2.4_5343o3hqgbg7dumghleddacmlq
-      '@itwin/core-bentley': 4.2.4
-      '@itwin/core-common': 4.2.4_y7bvomticisn7tqazrhygwcvia
-      '@itwin/core-electron': 4.2.4_hai52gjrnjoc5q34sxzm5bvjsa
-      '@itwin/core-frontend': 4.2.4_ndww6mrsjczj74w7dkh4mrygdi
-      '@itwin/core-geometry': 4.2.4
-      '@itwin/core-i18n': 4.2.4_@itwin+core-bentley@4.2.4
-      '@itwin/core-markup': 4.2.4_ikwwn3dmcp3el6lek5hvyfhjsi
-      '@itwin/core-orbitgt': 4.2.4
-      '@itwin/core-quantity': 4.2.4_@itwin+core-bentley@4.2.4
-      '@itwin/core-react': 4.7.0_hrkq4g5sabyxmzi3rxeqf4g4xe
-      '@itwin/core-telemetry': 4.2.4_@itwin+core-geometry@4.2.4
+      '@itwin/appui-abstract': 4.4.0_@itwin+core-bentley@4.4.0
+      '@itwin/appui-layout-react': 4.8.3_lfzg77bxt43jodc5fqdauyz7q4
+      '@itwin/appui-react': 4.9.0_crskxe37l4r5yja3undmbjlvka
+      '@itwin/components-react': 4.9.0_h76rwzigqohs2ex34373xit2f4
+      '@itwin/core-backend': 4.4.0_qs7o2gknk6xe34ewedxxsgg24e
+      '@itwin/core-bentley': 4.4.0
+      '@itwin/core-common': 4.4.0_cxsn5xen7eu56esbii6frxvzau
+      '@itwin/core-electron': 4.4.0_hx76q2zjw27w7jsyw5zbdbpk3y
+      '@itwin/core-frontend': 4.4.0_r2oyeiz6cvkfaphvsx2dlem3ea
```

Comment:
can you update lockfile to also pull in the 4.4.4 patch which you had a patch for dropping vps ?

Reply:
I could have sworn I ran `rush purge` then re ran `rush update` to bump up the lockfile, weird. I deleted pnpm-lock file manually and then ran rush update to make sure, and the 4.4.4 patch is in now.

Reply:
should avoid manually deleting lock file

use `rush update --full` next time